lib/generators/rodauth/templates/app/views/rodauth/tailwind/_login_form.html.erb

```diff
@@ -0,0 +1,47 @@
+            </div>
+          <% else %>
+            <div class="space-y-2">
+              <%= form.label "login", rodauth.login_label, class: "block text-sm text-gray-400" %>
+              <%= form.email_field rodauth.login_param, value: params[rodauth.login_param], id: "login", autocomplete: "email", required: true, class: class_names("w-full px-3 py-2 border rounded-md border-gray-400 dark:border-gray-700 dark:bg-gray-900 dark:text-gray-100 focus:dark:bg-gray-800 focus:dark:border-emerald-400 invalid:border-red-500 invalid:text-red-600 focus:invalid:border-red-500 focus:invalid:ring-red-500 valid:border-green-500", {"border rounded-md border-red-500": rodauth.field_error(rodauth.login_param)}), aria: ({ invalid: true, describedby: "login_error_message" } if rodauth.field_error(rodauth.login_param)) %>
+              <!-- up to here -->
+              <%= content_tag(:span, rodauth.field_error(rodauth.login_param), class: "text-red-800 text-xs mt-1 ml-1", id: "login_error_message") if rodauth.field_error(rodauth.login_param) %>
+            </div>
+          <% end %>
+          <% unless rodauth.skip_password_field_on_login? %>
+            <div class="space-y-2">
+              <div class="flex justify-between">
+                <%= form.label "password", rodauth.password_label, class: "block text-sm text-gray-400" %>
+                <%= link_to("Forgot password?", rodauth.reset_password_request_path, class: "text-xs text-blue-600 hover:text-blue-700 hover:underline dark:text-gray-200") %>
+              </div>
+              <%= form.password_field rodauth.password_param, value: "", id: "password", autocomplete: rodauth.password_field_autocomplete_value, required: true, class: class_names("w-full px-3 py-2 border rounded-md border-gray-400 dark:border-gray-700 dark:bg-gray-900 dark:text-gray-100 focus:dark:bg-gray-800 focus:dark:border-emerald-400 invalid:border-red-500 invalid:text-red-600 focus:invalid:border-red-500 focus:invalid:ring-red-500 valid:border-green-500", {"focus:border-red-400 focus:outline-none": rodauth.field_error(rodauth.password_param)}), aria: ({ invalid: true, describedby: "password_error_message" } if rodauth.field_error(rodauth.password_param)) %>
+              <%= content_tag(:span, rodauth.field_error(rodauth.password_param), class: "text-red-800 text-xs mt-1 ml-1", id: "password_error_message") if rodauth.field_error(rodauth.password_param) %>
+            </div>
+          <% end %>
+        </div>
+        <%= form.submit rodauth.login_button, class: "w-full px-8 py-3 font-semibold rounded-md 
+          w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-blue-600 hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 
+          dark:bg-emerald-400 dark:text-gray-900" %>
```

Comment:
I would prefer we consistently put all CSS class names in the same line, no matter how many they are. What do you think?

Reply:
Fixed: [fc54f92](https://github.com/janko/rodauth-rails/pull/114/commits/fc54f921fd6986ab2b0713602f9e72b54cec5a4c)